### PR TITLE
Start Private Message

### PR DESCRIPTION
**Pull request by cemuelle**
Possiblity to start new private message via long click on other messages on groups messaging.
It's also possible to start a new private message by clicking on the + icon on the top bar on the screen that display all private chats.
Also remove profil picture + name next to messages in private messaging

### CODE REVIEW

**Review by Svannia — COMMENTED** [review]
Looks good! The feature seems to work for me, as you said, should just review the end cases for having multiple private chats with the same user

**Comment by cemuelle** [comment]
The end case where we could start multiple new privates chats has been resolve

**Review by Ovoland — COMMENTED** [review]
Great pull review ! The chat seems to work perfectly on my tablet and the addition of private chat as well. As you can see on my different comments on the code, I have some recommendation concerning the UI. These visual changes could be done in a later PR if you're running out of time as they don't impact the app functionality. Good job !

File: app/src/main/java/com/github/se/studybuddies/ui/screens/ChatScreen.kt [hunk 1]
```diff
@@ -79,7 +80,7 @@ fun ChatScreen(viewModel: MessageViewModel, navigationActions: NavigationActions
     }
   }
 
-  OptionsDialog(viewModel, selectedMessage, showOptionsDialog, showEditDialog)
+  OptionsDialog(viewModel, selectedMessage, showOptionsDialog, showEditDialog, navigationActions)
   EditDialog(viewModel, selectedMessage, showEditDialog)
 
   Column(
```
Comment (Ovoland) [inline app/src/main/java/com/github/se/studybuddies/ui/screens/ChatScreen.kt:86]: Also try to match the general esthetic of the Figma and UI structure. Meaning try to use MainScreenScaffold for the chat screen. The 4 main screens should have a drawer menu and a bottom navigation bar.

**Comment by cemuelle** [comment]
Requested changes made

**Review by Ovoland — APPROVED** [review]
Good ! You took my comments in account and the private message UI is really nice as well as the backend. Really good job

**Pull request merged**